src/index.js

```diff
@@ -13,9 +13,9 @@ export default options => {
 
   /**
    * Get rooms socket is subscribed to.
-   *
-   * @param {String} id
-   * @param {Function} [fn]
+   * Used for get rooms
```

Comment:
Nahh!!! I would leave it as simple as possible, we don't even use JSDoc I had comments just for the sake of some clarity.